src/main/java/ddingdong/ddingdongBE/common/exception/CustomExceptionHandler.java

```diff
@@ -44,68 +47,71 @@ public ErrorResponse handleIllegalArgumentException(IllegalArgumentException exc
-
         loggingApplicationError(connectionInfo
                 + "\n"
-                + exception.getClass().getSimpleName() + " : " + message);
+                + BAD_REQUEST.value() + " : " + exception.message);
 
-        return new ErrorResponse(HttpStatus.BAD_REQUEST.value(), exception.getMessage(), LocalDateTime.now()
+        return new ErrorResponse(exception.errorCode, exception.message, LocalDateTime.now()
         );
     }
 
-    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
-    @ExceptionHandler(Throwable.class)
-    public ErrorResponse handleSystemException(Throwable exception, HttpServletRequest request) {
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
+    @ExceptionHandler(AuthenticationException.class)
+    public ErrorResponse handleAuthenticationException(AuthenticationException exception, HttpServletRequest request) {
         String connectionInfo = createLogConnectionInfo(request);
 
         loggingApplicationError(connectionInfo
                 + "\n"
-                + "[SYSTEM-ERROR]" + " : " + exception.getMessage());
+                + exception.getClass().getSimpleName() + " : " + exception.message);
 
-        return new ErrorResponse(HttpStatus.INTERNAL_SERVER_ERROR.value(), "Internal Sever Error", LocalDateTime.now()
+        return new ErrorResponse(exception.errorCode, exception.message, LocalDateTime.now()
         );
     }
 
-    @ResponseStatus(HttpStatus.UNAUTHORIZED)
-    @ExceptionHandler(AuthenticationException.class)
-    public ErrorResponse handleAuthenticationException(AuthenticationException exception, HttpServletRequest request) {
+    @ResponseStatus(BAD_REQUEST)
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ErrorResponse handleMethodArgumentNotValidException(MethodArgumentNotValidException exception,
+                                                               HttpServletRequest request) {
         String connectionInfo = createLogConnectionInfo(request);
 
+        String message = exception.getBindingResult().getFieldErrors().stream()
+                .findFirst()
+                .map(FieldError::getDefaultMessage)
+                .orElse("입력된 값이 올바르지 않습니다.");
+
         loggingApplicationError(connectionInfo
                 + "\n"
-                + exception.getClass().getSimpleName() + " : " + exception.message);
+                + exception.getClass().getSimpleName() + " : " + message);
 
-        return new ErrorResponse(exception.errorCode, exception.message, LocalDateTime.now()
+        return new ErrorResponse(BAD_REQUEST.value(), exception.getMessage(), LocalDateTime.now()
         );
     }
 
+
+    // TODO : NoSuchElementException 대신 PersistenceException.ResourceNotFound()로 전환 필요
     @ExceptionHandler(NoSuchElementException.class)
-    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ResponseStatus(BAD_REQUEST)
     public ExceptionResponse handleNoSuchElementException(NoSuchElementException e) {
```

Comment:
```suggestion
    public ExceptionResponse handleNoSuchElementException(NoSuchElementException exception) {
```
p3) 다른 부분과 통일되면 좋을 것 같아서 같이 수정해주시면 감사하겠습니다!

Reply:
컨텐츠와 헤더를 직접 세팅해줘야 해서 부득이하게 사용했습니다!